src/main/resources/io/jenkins/docker/connector/DockerComputerJNLPConnector/help-passSlaveConnectionArgs.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <p>
+        <span class="info">Prerequisites:</span> Docker image must derive from <a href="https://github.com/jenkinsci/docker-jnlp-slave/">jenkins/jnlp-slave</a>
```

Comment:
wrong. This isn't a MUST requirement. Just need java available on arbitrary image.
(also not required assuming you make "passSlaveConnectionArgs" the standard behaviour, without such an option)